language/src/regent/std.t

```diff
@@ -4518,10 +4524,10 @@ local function write_header(header_filename, registration_name)
   return registration_name, task_impl
 end
 
-function std.save_tasks(header_filename, filename, filetype, link_flags, registration_name)
+function std.save_tasks(header_filename, filename, filetype, link_flags, registration_name, task_whitelist)
```

Comment:
Can we change `task_whitelist` to be a map of the form `{ name = task_object }` where `name` is a string and `task_object` is a task?